src/phoenix/trace/span_evaluations.py

```diff
@@ -152,6 +163,55 @@ def __init_subclass__(
             tuple(sorted(prod)) for prod in product(*cls.index_names.keys())
         )
 
+    def to_parquet(self, directory: Optional[Union[str, Path]] = None) -> Path:
+        """Persists the evaluations to a parquet file.
+
+        Args:
+            directory (Optional[Union[str, Path]], optional): An optional path
+            to a directory where the parquet file will be saved. If not
+            provided, the parquet file will be saved to a default location.
+
+        Returns:
+            Path: The path to the parquet file, including a randomly generated
+            filename.
+        """
+        directory = Path(directory) if directory else TRACE_DATASET_DIR
+        path = directory / f"evaluations-{uuid4()}.parquet"
```

Comment:
I would probably move the uuid to be stable for the instance - otherwise you keep getting duplicates

Reply:
addressed in [0e52d9b](https://github.com/Arize-ai/phoenix/pull/2021/commits/0e52d9b1e05ce7941f117ba7b96e047648c2a0ea)